src/Microsoft.DotNet.Build.Tasks/AddDependenciesToProjectJson.cs

```diff
@@ -198,15 +195,32 @@ private void AddPackageItemsToDictionary(ref Dictionary<string, PackageItem> pac
             }
         }
 
-        // A versions file is of the form https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/master/Latest_Packages.txt
+        private static async System.Threading.Tasks.Task<string> GetFileAsync(string uri)
+        {
+            using (var stream = (File.Exists(uri)) ? File.OpenRead(uri) : await new HttpClient().GetStreamAsync(uri))
```

Comment:
I'm not sure File.Exists is the best way to do this check. See https://referencesource.microsoft.com/#System.Xml/System/Xml/XmlDownloadManager.cs,27 as an possible example to convert to a Uri and if it isn't a file then use HttpClient.